version.sbt

```diff
@@ -1 +1 @@
- ThisBuild / version := "0.12.1"
+ ThisBuild / version := "0.12.2"
```

Comment:
is this change big enough for a minor version bump?

Reply:
oh, meant to minor version bump, let me change